Titanium.Web.Proxy/Network/Tcp/TcpConnectionFactory.cs

```diff
@@ -149,5 +154,42 @@ internal class TcpConnectionFactory
                 Version = httpVersion
             };
         }
+
+        private bool IsLocalhost(string hostName)

```

Comment:
Can you please move it to Helper class.

https://github.com/justcoding121/Titanium-Web-Proxy/blob/develop/Titanium.Web.Proxy/Helpers/Network.cs#L34